### PR TITLE
Fix: Add tests for __unicode__ functions in Challenge.LeaderboardData model.

### PR DESCRIPTION
**Pull request by isht3**
Last of the PR for issue #1002

### CODE REVIEW

File: tests/unit/challenges/test_models.py [hunk 1]
```diff
@@ -115,25 +148,6 @@ class ChallengePhaseTestCase(BaseTestCase):
 
     def setUp(self):
         super(ChallengePhaseTestCase, self).setUp()
-        try:
```
Comment (TroJan) [inline tests/unit/challenges/test_models.py]: @isht3 : I am not able to understand why is this piece of code touched in this me PR. Can you please explain me ?

File: tests/unit/challenges/test_models.py [hunk 2]
```diff
@@ -175,34 +189,6 @@ class ChallengePhaseSplitTestCase(BaseTestCase):
 
     def setUp(self):
         super(ChallengePhaseSplitTestCase, self).setUp()
-        try:
```
Comment (TroJan) [inline tests/unit/challenges/test_models.py]: Same goes for here. Can you explain why is this removed ?

File: tests/unit/challenges/test_models.py (hunk 1 above)
Comment (isht3) [inline tests/unit/challenges/test_models.py]: @TroJan, Since that Class ChallengePhase Class was to be used to be in 3 TestClass', I thought it would be good to move it into BaseClass because it would remove redundancy and reduce number of lines of code.

**Comment by TroJan** [comment]
@isht3 : thanks for the fix. :)
